gradio/blocks.py

```diff
@@ -853,6 +859,11 @@ def set_event_trigger(
 
         rendered_in = LocalContext.renderable.get()
 
+        if js is True and inputs:
```

Comment:
I wonder if we can selectively remove this restriction for chatinterface and interface's internal ui update functions and make them run client side super fast

Reply:
We can do that in a follow up though

Reply:
Yeah I definitely want to use `js=True` in `gr.ChatInterface` and `gr.Interface` wherever possible, but I'll do it in a follow-up PR in a comprehensive way so we don't have to make any sort of exception